modules/ROOT/pages/get-started-install.adoc

```diff
@@ -20,8 +20,10 @@ include::partial$_set_page_context.adoc[]
 
 :downloads--url: https://www.couchbase.com/downloads/?family=sync-gateway[downloads page]
 :sg_download_link: pass:q,a[{url-package-downloads}/{version-maintenance}/]
+:sg_package_name_arm: couchbase-sync-gateway-community_{version-maintenance}_arm64
```

Comment:
Is this attribute used in the doc at all? No big deal if not

Reply:
No, it isn't. Created it as the doc didn't have an arm install package_name. Can be reused for other arm macos install.